simplePen.html

```diff
@@ -16,11 +16,11 @@
 
   <script type="text/paperscript" canvas="myCanvas">
 
-    var path;
-    
     // store paths created/removed for undo/redo purposes
     var pathsCreated = [];
     var pathsRemoved = [];
+
+    var numOfDivisions = 8;
```

Comment:
Take this value from the user by input. In the menu bar of the simple pen, you can give the user option to select this number. By default keep it 1 so that there is no symmetrical drawing unless user choose it

Reply:
Added slider for adjusting `numOfDivisions`, also added a checkbox to actually show users in which parts the canvas is divided.

Reply:
You can have a preview here https://tsarprince.github.io/Primitive-Paint/simplePen.html